modules/auction_manager/src/lib.rs

```diff
@@ -59,7 +59,9 @@ pub struct CollateralAuctionItem<AccountId, BlockNumber> {
 	/// current collateral amount for sale
 	#[codec(compact)]
 	amount: Balance,
-	/// Target sales amount want to get by this auction
+	/// Target sales amount want to get by this auction,
+	/// if zero, collateral auction will never be reverse stage,
+	/// otherwise, target amount is the actual payment amount of active bidder
```

Comment:
Consider the case which the target is zero, although it is unlikely to happen in the system. If target is zero, the collateral auction will never in reverse stage.